lib/davinci_dtr_test_kit/profiles/service_request_group.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'service_request/service_request_read'
+require_relative 'service_request/service_request_validation'
+
+module DaVinciDTRTestKit
+  class ServiceRequestGroup < Inferno::TestGroup
+    title 'CRD ServiceRequest Tests'
+    short_description 'Verify support for the server capabilities required by the CRD ServiceRequest Profile'
+    description %(
+      # Background
+
+    The CRD ServiceRequest sequence verifies that the system under test is
+    able to provide correct responses for ServiceRequest queries. These queries
+    must return resources conforming to the [CRD ServiceRequest Profile](http://hl7.org/fhir/us/davinci-crd/StructureDefinition/profile-servicerequest)
+    as specified in the DaVinci Coverage Requirements Discovery (CRD) v2.0.1
+    Implementation Guide.
+
+    # Testing Methodology
+    ## Read
+    First, Inferno will attempt to read each ServiceRequest resource id provided in
+    the ServiceRequest ids input. The server SHOULD be capable of returning a
+    ServiceRequest resource using the read interaction.
+
+    ## Profile Validation
+    Each resource returned from the read step SHALL conform to
+    the [CRD ServiceRequest Profile](http://hl7.org/fhir/us/davinci-crd/StructureDefinition/profile-servicerequest).
+    Each element is checked against terminology binding and cardinality requirements.
+
+    )
+    id :service_request_group
+    run_as_group
+
+    input :service_request_ids,
+          title: 'Service Request IDs',
+          description: 'Comma separated list of ServiceRequest IDs'
```

Comment:
Since support for ServiceRequests is not required, let's make the input optional, and skip the ServiceRequest tests when not present. Similar for other non-QR profiles.

Reply:
Addressed in [Make tests and id inputs optional](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35/commits/c11b7ce7a3ad9abafa26cad57c260f2930074a0a)